robottelo/constants.py

```diff
@@ -797,8 +797,6 @@
     'Edit hosts',
     'Edit partition tables',
     'Red Hat Access Logs',
-    'Remote Execution Manager',
-    'Remote Execution User',
```

Comment:
Hmm, I can see interesting difference for RHEL 6 and 7.
RHEL7:
![image](https://cloud.githubusercontent.com/assets/10895065/15603942/5092fb96-2406-11e6-93b5-f731cc5dc38b.png)
RHEL6:
![image](https://cloud.githubusercontent.com/assets/10895065/15603961/68ed53d0-2406-11e6-983c-374c0c055b79.png)

Is it intended behaviour?